### DISCUSSION TITLE
Well Correlation Tool - 't21-hack-wellCorr'

### DISCUSSION THREAD

**Discussion by mlbelobraydi** [softwareunderground/transform-2021-hackathon#12] [Projects]
## Slack channel
[t21-hack-wellCorr](https://swung.slack.com/archives/C01TSKT44AU)

## People
* Maria Cecilia Bravo (champion)
* Matthew Belobraydic
* Michael Harty

## Description
Open source manual well correlation tools are nearly non-existent. Well correlations are the basic interpretations methodology to test stratigraphic and structural hypotheses. These correlations can assist with seismic depth conversion and petrophysical zonation for more advanced interpretations, along with providing inputs for maps of structure and thickness. Without a way to provide expert well correlations on well log data, it is difficult to support other projects on SWUNG or amass enough clean and validated data for later ML processes. 
*GitHub repository: https://github.com/mlbelobraydi/t21-hack-wellcorr (just started)
*Initial project Kanban: https://github.com/mlbelobraydi/t21-hack-wellcorr/projects/1

## Goals
* Build a method that can allow for manual well correlation manipulation for multiple tops for multiple wells at once
* Have a way to store, recall, and extract well correlations for display upon a well log display or export
* Identification of pain-points and technology development for potential future work past the hack-a-thon
* example: https://files.slack.com/files-pri/T094HBB9T-F01SSRWCQH3/widget3.gif

## Resources
* Currently undetermined

## Skills needed (if any)
If you're looking for teammates with particular skills, list them here.
We are looking for help and assistance in any and all domains.
* Geoscience
* Programming
* Front-end / JS
* Databases
* Cloud
* Machine learning
* Testing
* Documentation

**Comment by filippo82** [comment]
[T21 Hackathon Idea Mashup](https://softwareunderground.org/events/2021/4/7/t21-hackathon-idea-mashup) today (or tomorrow depending on your time zone) Wednesday April 14th at 16:00 UTC. Come to Slack to find the link for the meeting!

**Comment by mlbelobraydi** [comment]
We are using Google Slides to bodyshop some of the features we want to build: [Google Slides](https://docs.google.com/presentation/d/1Tuu-TL_DTL85YHTApIHH3we85x6Qe7v2NUo67wLvCgA/edit#slide=id.gcde51a2159_0_96)
